File: Algoritmos Traduzidos/Perl/Algoritmos_Filipe_Dias/110.pl
print "Escreva o valor de a:\n";
$a = <STDIN>;
print "Escreva o valor de b:\n";
$b = <STDIN>;
if ($a != 0)
{
	$x = ($b * -1)/$a;
	print "O valor de x e $x\n";
}
else
{
	print "Nao existe zero.";
}

File: Algoritmos Traduzidos/Ruby/Algoritmos_Filipe_Dias/110.rb
puts "Escreva o valor de a:"
a = gets
print "Escreva o valor de b:"
b = gets
if a.to_f != 0
  x = (b.to_f * -1)/a.to_f
  puts "O valor de x e #{x}"
else
  puts "Nao existe zero."
end